### PR TITLE
add "write to eth" cookbook

### PR DESCRIPTION
**Pull request by bitful-pannul**

### CODE REVIEW

**Review by dr-frmr — APPROVED** [review]
LGTM!

File: src/cookbook/writing_data_to_eth.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        vm.startBroadcast(wallet.privateKey);
+        
+        Counter counter = new Counter();
+        console.log("Counter deployed at address: ", address(counter));
+        vm.stopBroadcast();
+    }
+}
+```
+
+Now we boot a fakechain, either with `kit f` which boots one at port 8545 in the background, or manually with `anvil --load-state kinostate.json`.
```
Comment (nick1udwig) [inline src/cookbook/writing_data_to_eth.md:61]: Put lines that are for copy-pasting in between triple backtics. I.e. this line should probably read

Now we boot a fakechain, either with `kit f` which boots one at port 8545 in the background, or manually with 
```
anvil --load-state kinostate.json
```

File: src/cookbook/writing_data_to_eth.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+Then we can run `forge script --rpc-url http://localhost:8545 script/Deploy.s.sol --broadcast` we'll see a printout that looks something like this:
```
Comment (nick1udwig) [inline src/cookbook/writing_data_to_eth.md:63]: Same as above: to triple backtics.

File: src/cookbook/writing_data_to_eth.md [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+        }
+    
+        function increment() public {
+            number++;
+        }
+    }
+}
+```
+
+Pretty cool huh? We can now do things like define a setNumber() call just like this:
```
Comment (nick1udwig) [inline src/cookbook/writing_data_to_eth.md:110]: One sentence per line

File: src/cookbook/writing_data_to_eth.md [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+            Ok(()) => {}
+            Err(e) => {
+                println!("error: {:?}", e);
+            }
+        };
+    }
+}
+```
+
+Now, we'll add the 2 writes that are possible: increment() and setNumber(newNumber). To do this, we'll need to define a wallet, and import a few new crates:
```
Comment (nick1udwig) [inline src/cookbook/writing_data_to_eth.md:185]: One sentence per line

File: src/cookbook/writing_data_to_eth.md [hunk 5]
```diff
@@ -0,0 +1,310 @@
...
+        let signed = TxEnvelope::from(tx.into_signed(sig));
+        let mut buf = vec![];
+        signed.encode(&mut buf);
+
+        let tx_hash = provider.send_raw_transaction(buf.into());
+        println!("tx_hash: {:?}", tx_hash);
+    }
+```
+
+Nice! Putting it all together, we can build and start the package on a fake node (`kit f` if you don't have one running), `kit bs`.
```
Comment (nick1udwig) [inline src/cookbook/writing_data_to_eth.md:294]: One sentence per line

**Review by nick1udwig — CHANGES_REQUESTED** [review]
Overall comment: style is to use `you`s and no `we`s or `let's`. I know it is quite natural to write prose like you do here, but we need to keep consistent style across the docs. Please rewrite to remove `we`s and `let's`.

**Pull request merged**